modules/rostests/apitests/user32/ScrollBarRedraw.c

```diff
@@ -0,0 +1,727 @@
+/*
+ * PROJECT:     ReactOS api tests
+ * LICENSE:     ISC (https://spdx.org/licenses/ISC)
```

Comment:
Any chance to license under GPL/LGPL/MIT/something we already use? I found no cases of this license mentioned in the headers in our source tree.

Reply:
Sure!  The MIT license is the most similar one to the ISC license that I like to release my code under, so I've switched it over to that.